docs/sharing.md

```diff
@@ -1,41 +1,47 @@
+webKnossos is built for working collaboratively and sharing your work with colleagues, collaborators, and the scientific community.
+Anything in webKnossos can be shared: the raw datasets, any skeleton or volume annotations, or complex segmentations.
 
-To manage access rights to certain datasets for webKnossos users check out [the Datasets guide](./datasets.md#general).
+When speaking about collaboration and sharing, we imagine two scenarios supported by webKnossos:
+1. Sharing data with outsiders - anyone who is not a member of your webKnossos organization, e.g., colleagues from other institutes, reviewers, publishers, a paper publication, and the research community as a whole.
+2. Share data within your organization to collaborate with other members/co-workers
 
-## Dataset Sharing
+Since webKnossos is a web platform, most resources can be shared as a web link/URL. This makes it super easy to integrate webKnossos in your existing communication workflows, e.g.:
+- send collaborators an *email* containing a link to specific, interesting data location or annotations
+- include a link in a *publication* so readers can have direct access to the data to see for themselves
+- share a link through *Slack*, *MS Teams*, or any other messenger service
+- include it in any *lab journal* and *blog* posts
+- link it in a *forum* post
 
-Dataset sharing allows outside users to view your datasets and segmentation layers within webKnossos.
-Shared resources can be accessed through a direct URL or can be featured on a spotlight gallery for showcasing your work.
-[Please contact us](mailto:hello@webknossos.org) to feature your dataset on https://webknossos.org.
+In many ways, the sharing by web link works similarly to products like Google Drive or Dropbox.
 
-Sharing a dataset is useful for multiple scenarios: 
-- You recorded a novel microscopy dataset and want to include links to it in your paper or for reviewers. Use wklink.org to shorten these URLs, e.g. https://wklink.org/5386 ([contact us](mailto:hello@webknossos.org)).
-- You created an interesting, highly-accurate segmentation layer for an existing dataset and want to share it for your publication.
-- You have worked and published several datasets over the years and want to have a single gallery for all your public datasets.
+webKnossos sharing is tightly integrated with user permissions and access rights. See the sections [on dataset management](./datasets.md#general) and [user administration](./users.md) for more info.
 
-webKnossos shares datasets publicly (everyone can view them without any login) or privately (a login or special URLs are required to view the link).
+## Sharing 
+As mentioned earlier, any webKnossos resource can be shared as a normal web URL. The easiest way to obtain this sharing link is by clicking the `Share` button in the toolbar at the top of the screen next to the position/coordinate section (*Share icon*). 
```

Comment:
I started with this section thinking dataset and annotation sharing work the same way... Then I gave up... might want to remove this section of merge it with other stuff.

Reply:
This section does not describe dataset sharing. These options are not available for datasets. only for annotations. its confusing.

Reply:
But what is this section for? There is another "Annotation sharing" section